peps/pep-0772.rst

```diff
@@ -0,0 +1,1612 @@
+While its basic functionality is okay, interference may come from many sources,
+such as virus scanners, other installers, system configuration, admin policies,
+and even other files in the same directory as the installer. On top of this,
+most of its advanced and beneficial functionality such as update patches,
+incremental updates, and automatic rollback are unimportant for Python users.
+
+Most user expectations are *defined* by the traditional installer, and so by
+definition, it meets them. One primary gap is that it is not able to create an
+"unmanaged" install - that is, the equivalent of only copying files onto the
+user's system without registration. If you have installed it once, and you
+try to install it again, you will only even be able to manage (or upgrade) the
+existing install. This can lead to installs moving on update, which will
+break users.
+
+Additionally, the ``PATH`` environment variable cannot be intelligently
+modified - at best, we can prepend or append the install path. This usually
+results in the most recent install of Python being the highest priority. For
+example, if the user has Python 3.14 installed and then installs (or updates)
+3.13, the ``python`` command will switch from the later version to the earlier
+version.
+
+The ``py.exe`` launcher, defined in PEP 397 and implicitly updated by PEP 514,
+is an attempt to avoid this particular issue. It uses its own logic for finding
+installed versions without relying on ``PATH``. However, the PEP 514 logic does
+not allow for prerelease or experimental builds to be treated specially, and so
+``py.exe`` often prefers these builds by default over the non-experimental
+version expected by the user.
+
+The Windows Store package is very reliable, with the exception of the global
+shortcuts. Rather than modifying ``PATH`` to add its own directory, these
+shortcuts are created in a single OS managed directory that has all the
+shortcuts defined by any app. Users are able to modify their ``PATH`` to exclude
+or de-prioritise this directory, leading to unreliable or inconsistent
+behaviour, and historically we have also seen this caused by installers.
+For example, installing Python from the Store followed by Python from the
+traditional installer with its ``PATH`` modification enabled will almost always
+shadow the Store package's Python with the later install.
+
+User expectations that are un-met by the Store package tend to be performance
+and technical. Due to the overhead of launching an app, Python starts up slower.
+Because apps are designed to be isolated from each other, it is more difficult
+to use hidden directories (such as ``AppData`` or ``TEMP``) to communicate
+between different versions of Python, as each version has its own space. Apps
+are subject to stricter security requirements that legacy applications start
+disabled, such as DLL hijacking protection, which causes some libraries to fail.
```

Comment:
"that legacy applications start disabled" reads oddly, I'm not sure what is intended here.